app/tests/v1/test_question.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+import json
+
+from app import create_app
+
+class TestQuestions(unittest.TestCase):
+
+    def setUp(self):
+        self.app = create_app("testing")
+        self.client = self.app.test_client()
+
+    def create_record(self):
+        response = self.client.post('/api/v1/questions', 
+            data=json.dumps({
+                "Title": "Javascript",
+                "question" : "How do you write Promises?"
+                }),
+            headers={"content-type": "application/json"})
+        return response
+
+    ###Test for creation of questions
+    def test_01_post_questions(self):
+        response = self.create_record()
+        self.assertEqual(response.status_code, 201)
```

Comment:
Add end of file new line here

Reply:
Thanks for that. I had to read up on why ending a file with a new line is important. I'll have to look through all my code and make changes to all my files